_drafts/zero-to-main-libc.md

```diff
@@ -0,0 +1,721 @@
+---
+title: "From zero to main(): Bootstrapping libc with Newlib"
+description:
+  "TODO"
+author: francois
+tags: [zero-to-main]
+---
+
+In this series of post, we’ve worked methodically to demystify what happens to
+firmware before the main() function is called. So far, we bootstrapped a C
+environment, wrote a linker script from scratch, and implemented our own
+bootloader.
+
+And yet, we cannot even write a hello world program! Consider the following
+`main.c` file:
+
+```c
+#include <stdio.h>
+
+int main() {
+  printf("Hello, World\n");
+  while (1) {}
+}
+```
+Compiling this using our makefile and linker script from [previous](% post_url
```

Comment:
Link is broken 

`[post]({% post_url 2019-09-17-continuous-integration-for-firmware %})`

Reply:
👍